chain-storage/Cargo.toml

```diff
@@ -11,7 +11,9 @@ edition = "2018"
 
 [dependencies]
 chain-core = { path = "../chain-core" }
+rand = "0.6"
 
 [dev-dependencies]
+#rand = "0.6"
```

Comment:
Just remove this line.

Reply:
Actually, if `rand` is only used in test code, the reorganization described in my previous comment would allow it to be kept as a dev-dependency.